webpack.prod.js

```diff
@@ -0,0 +1,62 @@
+const path = require("path");
+const HTMLWebpackPlugin = require("html-webpack-plugin");
+const TerserPlugin = require("terser-webpack-plugin");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const { CleanWebpackPlugin } = require("clean-webpack-plugin");
+const Dotenv = require('dotenv-webpack');
+
+module.exports = {
+  mode: "production",
+  entry: "./src/index.js",
+  output: {
+    path: path.join(__dirname, "/dist"),
+    filename: "bundle.js",
```

Comment:
Add hashing to invalidate browser cash automatically when new changes arrive (https://webpack.js.org/guides/caching/)

Reply:
added hashing in webpack.prod.js